PyRDF/__init__.py

```diff
@@ -55,3 +55,6 @@ def include(includes_list):
         includes_list = [includes_list]
 
     includes.extend(includes_list)
+
+def broadcastInitialization(fun, *args, **kwargs):
```

Comment:
Now in the right place :)

Since this is the API function, I would prefer to call it something shorter like `initialize`.

Reply:
And we need documentation for this function.